src/commands/flow/internal/allFlowsCompleted.ts

```diff
@@ -0,0 +1,34 @@
+export const allFlowsCompleted = (
+  flows: any
+): {
```

Comment:
it would be helpful to have defined return type

Reply:
fixed